### PR TITLE
Made some changes to Light theme

### PR DESCRIPTION
**Pull request by DanielNugent**
Light theme UI Improvements

## Purpose

The light theme needed a bit of reworking + some general improvements

## Proposed Change

Improving UI on light theme as the markers might not have use dark mode.

(Ill add some screenshots now)

### CODE REVIEW

File: screens/ProfileScreen.tsx [hunk 1]
```diff
@@ -270,7 +269,11 @@ export default function ProfileScreen({ navigation }: RootTabScreenProps<'Profil
           </View>
         </Modal>
         <View style={styles.coverRegion}>
-          <ImageBackground resizeMode="cover" source={coverPic || { uri: undefined }} style={styles.coverImage}>
+          <ImageBackground
+            resizeMode="cover"
+            source={coverPic || { uri: 'https://i.imgur.com/WA0B7eL.png' }}
```
Comment (DanielNugent) [inline screens/ProfileScreen.tsx:274]: add a default image for no cover photo

File: screens/ProfileScreen.tsx [hunk 2]
```diff
@@ -478,6 +479,8 @@ const styles = StyleSheet.create({
     // backgroundColor: 'white',
     borderWidth: 5,
     borderColor: '#30363d',
+    flex: 1,
```
Comment (DanielNugent) [inline screens/ProfileScreen.tsx:482]: extend the "instagram" image square grid background to the bottom of the screen

File: screens/StyleFolderModal.tsx [hunk 3]
```diff
@@ -509,7 +509,6 @@ const styles = StyleSheet.create({
     flex: 1,
     height: '100%',
     paddingVertical: 10,
-    backgroundColor: 'rgb(51,51,51)',
```
Comment (DanielNugent) [inline screens/StyleFolderModal.tsx:512]: this didnt look that great in light theme

**Review by ClaireCassidy — APPROVED** [review]
Looks great!

**Review by lexesj — APPROVED** [review]
Looking good!

**Pull request merged**